doc/cross-compile/linux-arm.md

```diff
@@ -0,0 +1,243 @@
+# Cross-compiling iceoryx2
+
+## Cross-compile project with `cross`
+
+Setup `cross` as described here: [cross-rs GitHub](https://github.com/cross-rs/cross).
+
+Once `cross` is setup, navigate to root of the iceoryx2 repo and run
+the command below.
+
+```bash
+cross build --target aarch64-unknown-linux-gnu --release --package iceoryx2-ffi
```

Comment:
Does this work?

From the discussion in the issue I assumed you needed to use the `--features libc_platform`?

Reply:
Okay, then we could also just leave it out or add a note that there is this feature flag is someone has issues.

There is often an issue with bindgen when the feature flag is not used and some user complain about the libclang dependency.